src/Lykke.MatchingEngine.Connector/ConvertExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Lykke.MatchingEngine.Connector.Models;
+
+namespace Lykke.MatchingEngine.Connector
+{
+    public static class ConvertExtensions
+    {
+        public static Fee ToApiModel(this Domain.Fee src)
+        {
+            return new Fee
```

Comment:
Consider to add ctor or static factory method with parameters to the Fee class and hide public setters. It will prevent of missed field in mappings.